templates/linux/squeak.sh

```diff
@@ -4,14 +4,54 @@
 # Version:     2.4
 # Date:        2020/03/31
```

Comment:
Please update the file header (add yourself to authors, bump version, and update date).